src/ocl/convolutionocl.cpp

```diff
@@ -140,6 +253,122 @@ static void ShrinkToFind10Results(std::vector<Solution>& found)
     found = std::move(out);
 }
 
+std::vector<Solution> VerifiedFDBSolution(const ExecutionContext& ctx,
```

Comment:
The main change for this find mode is within this function and in the search enforcement on lines 449, 450. @randyspauldingamd the unit testing of VerifiedFDBSolution might be sufficient for this find mode addition. Though this is complicated by the reliance on db paths which are not programmatically settable. Alternatively we might reduce this into smaller functions which lend themselves better to testing.